### PR TITLE
chore: add instructions on how to clear electron cache

### PR DESCRIPTION
**Pull request by slowbackspace**
I always end up googling these paths :)

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -18,16 +18,33 @@
...
+`/Users/<user>/Library/Application Support/@trezor/suite-desktop/IndexedDB`
+
+### Clearing Electron cache
+To clear electron cache delete following folder:
+
+#### Windows
+`C:\Users\<user>\AppData\Roaming\@trezor\suite-desktop\Cache`
+
+#### Linux
+`/home/<user>/.config/@trezor/suite-desktop/`
```
Comment (prusnak) [inline NOTES.md]:
```suggestion
`/home/<user>/.config/@trezor/suite-desktop/Cache`
```

File: NOTES.md [hunk 2]
```diff
@@ -18,16 +18,33 @@
...
+
+#### macOS
+`/Users/<user>/Library/ApplicationSupport/@trezor/suite-desktop/`
```
Comment (prusnak) [inline NOTES.md]:
```suggestion
`/Users/<user>/Library/ApplicationSupport/@trezor/suite-desktop/Cache`
```

**Review by prusnak — CHANGES_REQUESTED** [review]

File: NOTES.md (hunk 2 above)
Comment (slowbackspace) [reply NOTES.md]: huh thanks! should be all good now. Sorry about the force-push, but I don't want to mess around with fix-ups for such a small pr.

**Comment by mroz22** [comment]
I vote for moving this into docs folder. So that all readmes are in one place.

**Comment by slowbackspace** [comment]
I've moved the file to docs folder although I think that the documentation needs more love and consolidation as it is still all over the place. Refactoring this was not really the intention of original PR, just fixing some paths that are already there ~but are incorrect.~ eh seems windows path was completely missing.

**Review by keraf — APPROVED** [review]

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**